flight/Modules/Airspeed/airspeed.c

```diff
@@ -104,10 +104,10 @@ static int8_t airspeedADCPin=-1;
 // Private functions
 static void airspeedTask(void *parameters);
 void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime, uint8_t airspeedSensorType, int8_t airspeedADCPin);
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev);
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len);
```

Comment:
Good idea. That would resolve my two comments regarding the ```NULL, NULL, NULL, 0``` pattern and the ```(void) ev; (void) ctx; (void) obj; (void) len;``` declaration.